stockroom/diff.py

```diff
@@ -0,0 +1,126 @@
+from typing import NamedTuple
+
+from hangar import Repository
+from rich.jupyter import print
+from rich.table import Table
+from rich.segment import Segment
+from rich.console import Console
+
+from .parser import tagkey, SEP
+
+class ParsedDiff(NamedTuple):
+    tagName: str
+    ref_sample: str = ''
+    dest_samples: list = list()
+
+class TagDiff:
+    """
+    The Diff object for tags
+    """
+
+    def __init__(self, ref_commit: str = None, dest_commit: str = None,
+                    root: str = None):
+        self.ref_commit = ref_commit
+        self.dest_commit = dest_commit
+        self.repo = Repository(root)
+        self.all_keys = set()
+
+        self._added: dict = dict()
+        self._deleted: dict = dict()
+        self._mutated: dict = dict()
+
+        self.parse_diff()
+
+    def gen_tables(self):
+    """
+    Generate the Rich Table using parsed data.
+    """
+
+        table = Table(title='Diff %s..%s'%(self.ref_commit[2:7], self.dest_commit[2:7]))
+
+        # Generate Columns
+        column_headings = ['Tags'] + [self.ref_commit, self.dest_commit]
```

Comment:
Can't you just do `['Tags', self.ref_commit, self.dest_commit]`

Reply:
Yeah this seems better...will convert